### PR TITLE
Added Nested Classes to ClassFileBuilderTest

### PR DESCRIPTION
**Pull request by JuliusNmn**
Test runs fine

### CODE REVIEW

File: OPAL/ba/src/test/scala/org/opalj/ba/ClassFileBuilderTest.scala [hunk 1]
```diff
@@ -89,19 +89,46 @@ class ClassFileBuilderTest extends AnyFlatSpec {
             attributes = RefArray(permittedSubclassesAttribute, br.Synthetic)
         ).toDA()
 
+    val (nestedClassInner, _) =
+        CLASS[Nothing](
+            version = UShortPair(0, 49),
```
Comment (errt) [inline OPAL/ba/src/test/scala/org/opalj/ba/ClassFileBuilderTest.scala]: Is this the proper Java version? Also, I suggest we move this upwards so features introduced later (i.e., with a higher version number here) are later in this file.

**Review by errt — COMMENTED** [review]
LGTM, but please address comment.

**Comment by JuliusNmn** [comment]
The initial pull request did not test the NestHost attribute. This has been added.

**Review by errt — CHANGES_REQUESTED** [review]
I think the class file versions are still wrong. Also, the CI failed in formatting, so please run `sbt scalariformFormat`.

**Review by errt — CHANGES_REQUESTED** [review]
LGTM, just please reorder the code so that features are handled in order of their addition to the JVM (i.e., by class file version). Once that is done, I'll approve this PR.

**Review by JuliusNmn — COMMENTED** [review]
classes have been ordered by class file version

**Review by errt — DISMISSED** [review]

**Review by errt — APPROVED** [review]

**Pull request merged**